.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: APK Release
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+      - v[0-9]+.[0-9]+.[0-9]+-[0-9a-zA-Z]+
+      - v[0-9]+.[0-9]+.[0-9]+-[0-9a-zA-Z]+-[0-9a-zA-Z]+
+env: 
+ CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }} 
+ KEYSTORE_ALIAS: ${{ secrets.KEYSTORE_ALIAS }}
+ KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
+ KEYSTORE_PASSWORD: ${{ secrets.KEYSTORE_PASSWORD }}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel previous workflow runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+          
+      - name: Decode Keystore file
+        run: echo $ENCODED_KEYSTORE | base64 -di > "${HOME}"/ona.keystore.jks
+        env:
+          ENCODED_KEYSTORE: ${{ secrets.KEYSTORE_FILE }}  
+
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: Decode & Generate keystore.properties file
+        run: echo $KEYSTORE_PROPERTIES | base64 -di > keystore.properties
+        env:
```

Comment:
Can we also add the `oauth.client.id` and `oauth.client.secret` to the `local.properties` file?